presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -112,6 +112,12 @@ public void destroy()
         connectionFactory.close();
     }
 
+    @Override
+    public boolean schemaExists(String schema)
```

Comment:
This could go as the default impl in `JdbcClient` interface

Reply:
Good idea, will move there